src/components/MultiHopTrade/hooks/useGetTradeQuotes/useGetTradeQuotes.tsx

```diff
@@ -215,18 +214,14 @@ export const useGetTradeQuotes = () => {
         const receiveAccountNumber = receiveAssetBip44Params?.accountNumber
 
         const tradeAmountUsd = bnOrZero(sellAssetUsdRate).times(debouncedSellAmountCryptoPrecision)
-        const potentialAffiliateBps = DEFAULT_SWAPPER_AFFILIATE_BPS
-        const affiliateBps = (() => {
-          // free trades if there's an error getting foxHeld
```

Comment:
Shouldn't still be present? This is not a bug, it's part of the flow - failure to get foxHeld means free trades

Reply:
solved in 2d96688c3c4f8e48caed1267aa2734dd8d06eb1b with unit test